libpod/kube.go

```diff
@@ -119,6 +119,61 @@ func (p *Pod) getInfraContainer() (*Container, error) {
 	return p.runtime.GetContainer(infraID)
 }
 
+func GenerateForKubeDaemonSet(ctx context.Context, pod *YAMLPod, options entities.GenerateKubeOptions) (*YAMLDaemonSet, error) {
+	// Restart policy for DaemonSets can only be set to Always
+	if options.Type == define.K8sKindDaemonSet && !(pod.Spec.RestartPolicy == "" || pod.Spec.RestartPolicy == v1.RestartPolicyAlways) {
+		return nil, fmt.Errorf("k8s DaemonSets can only have restartPolicy set to Always")
```

Comment:
I don't see this specified in the man page and it should be.

Reply:
Ok, I didn't find this for other types which I guess is probably due to a lack of --restartPolicy flag where this can be clearly documented. What do you think about documenting this for both Deployments and DaemonSets under "Description" in the man page?

Reply:
SGTM

Reply:
Done